lib/src/argument_list_visitor.dart

```diff
@@ -213,7 +211,7 @@ class ArgumentListVisitor {
       }
 
       _visitor.builder.startSpan();
-      _argumentsAfterFunctions.visit(_visitor);
+      _argumentsAfterFunctions!.visit(_visitor);
```

Comment:
Consider adding an assertion to the constructor to justify why this is safe, e.g.:

```dart
  ArgumentListVisitor._(
      this._visitor,
      this._leftParenthesis,
      this._rightParenthesis,
      this._allArguments,
      this._arguments,
      this._functions,
      this._argumentsAfterFunctions)
  : assert(_functions == null || _argumentsAfterFunctions != null,
        'If _functions is specified, _argumentsAfterFunctions should be too');
```

Reply:
Done.